internal/flags/gitlab_visibility.go

```diff
@@ -0,0 +1,41 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/fluxcd/flux2/v2/internal/utils"
+)
+
+var supportedGitLabVisibilities = []string{
+	"public",
+	"internal",
+	"private",
+}
+
+type GitLabVisibility string
```

Comment:
Fixed in c032ef991ddd9f7914b1f82c5bb097100c0dfe9b

Reply:
@darkowlzz I thought about using `ValidateRepositoryVisibility`. Doing it would mean that when a new `RepositoryVisibility` is added to Flux, GitLab will automatically accept it as valid, while it might not be.

Reply:
Refactored in 047b51a83377bb82abd9253e8289177ef9c03190